### PR TITLE
Fixed macOS shortcut

### PR DESCRIPTION
**Pull request by RayFV**
correction macOS's shortcut

### CODE REVIEW

File: getting_started/step_by_step/instancing.rst [hunk 1]
```diff
@@ -88,7 +88,7 @@ Click on it and drag it towards the center of the view.
 
 .. image:: img/instancing_ball_moved.png
 
-Play the game by pressing :kbd:`F5` (:kbd:`Cmd + B` on macOS). You should see it fall.
+Play the game by pressing :kbd:`F5` (:kbd:`Cmd + R` on macOS). You should see it fall.
```
Comment (AThousandShips) [inline getting_started/step_by_step/instancing.rst:91]: This is incorrect, the setting is `Cmd + B` according to the settings, have you changed it in yours?

Comment (AThousandShips) [reply getting_started/step_by_step/instancing.rst:91]: You can however change the reference to `F5` below to also include the macOS key

Comment (RayFV) [reply getting_started/step_by_step/instancing.rst:91]: You are right. I was probably on the `Script` tab and `Cmd + B` is not working that make me confuse.

**Review by AThousandShips — APPROVED** [review]

**Comment by AThousandShips** [comment]
(Squashed the commits into one)

**Review by skyace65 — APPROVED** [review]

**Comment by skyace65** [comment]
Thanks! And congrats on your first merged PR!

**Comment by mhilbrunner** [comment]
Cherry-picked to 4.1.

**Pull request merged**